tests/integrations/realcluster/reboot_pd_test.go

```diff
@@ -26,6 +27,8 @@ import (
 func restartTiUP() {
 	log.Info("start to restart TiUP")
 	cmd := exec.Command("make", "deploy")
+	cmd.Stdout = os.Stdout
```

Comment:
What is the current output?

Reply:
`exec.Command` creates a child process, so it will output on the child process's terminal.